packages/headless/src/features/facets/range-facets/date-facet-set/interfaces/request.ts

```diff
@@ -27,7 +33,19 @@ export interface DateRangeRequest {
   state: FacetValueState;
 }
 
+export interface DateRangeMappedRequest extends DateRangeRequest {
```

Comment:
Used to pass the values from the action to the slices.
Will also be used to return the search parameters

Reply:
Would be simpler to extend `DateRangeApiRequest` instead of `DateRangeRequest`, and add keys to that interface? e.g.

```
export interface DateRangeMappedRequest extends DateRangeApiRequest {
    relativeStart?: RelativeDate;
    relativeEnd?: RelativeDate;
}
```